App/reducersEZ.ts

```diff
@@ -1,29 +1,16 @@
 /// <reference path="../typings/tsd.d.ts" />
 
 import { Reducer, combineReducers } from 'redux';
-import { IEditText } from './actionsEZ';
+import { IChangeFollowState } from './actionsEZ';
+import { InitialState, IWorkItem } from './statesEZ';
 
-const InitialState: sampleState = {
-  value: 'initialstate',
-  foo: true
-}
-
-export interface sampleState {
-  value: string;
-  foo: boolean;
-}
-
-function sampleReducer(state: sampleState = InitialState, action: IEditText): sampleState {
```

Comment:
Same as above, I named it something more informative in the commit and removed this part.